src/app/data-catalogue/components/data-catalogue-detail-page/data-catalogue-detail-page.component.html

```diff
@@ -12,11 +12,13 @@
       </a>
       <div class="data-catalogue-detail-page__title" [ngClass]="{'data-catalogue-detail-page__title--mobile': screenMode === 'mobile'}">
         @if (baseMetadataInformation.imageUrl) {
-          <img
-            [src]="baseMetadataInformation.imageUrl"
-            class="data-catalogue-detail-page__title__image"
-            alt="Beispielbild zur Karte {{ baseMetadataInformation.itemTitle }}"
-          />
+          <a [href]="baseMetadataInformation.imageUrl.url.href">
+            <img
+              [src]="baseMetadataInformation.imageUrl.src.href"
```

Comment:
Can't we use `ngSrc` here? Is it because we don't know the images' sizes?

Reply:
Image Sizes can vary, we only set a max width